### PR TITLE
LPS-98893 Add MappedContent component

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by p2kmgcl** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/state.es.js [hunk 1]
```diff
@@ -475,6 +475,12 @@ const INITIAL_STATE = {
 	 */
 	mappedAssetEntries: Config.array().value([]),
 
+	/**
+	 * @default []
+	 * @review
+	 * @type {object[]}
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/state.es.js]: Array of what?

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/sidebar/page_content/MappedContents.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		<ul className="list-unstyled">
+			{[...props.mappedContents].map(mappedContent => (
+				<MappedContent key={mappedContent.classPK} {...mappedContent} />
+			))}
+		</ul>
+	);
+};
+
+MappedContents.propTypes = {
+	mappedContents: PropTypes.array
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/sidebar/page_content/MappedContents.js]: Array of what?
Write at least the `classPK` property

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/state.es.js [hunk 3]
```diff
@@ -475,6 +475,12 @@ const INITIAL_STATE = {
...
+	 */
+	mappedContents: Config.array().value([]),
 	/**
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/state.es.js]: Missing new line

**Comment by p2kmgcl** [comment]
Pull request submitted to https://github.com/ealonso/liferay-portal/pull/2484. [See changes here.](https://github.com/p2kmgcl/liferay-portal/compare/7c777ff6e745~2...7c777ff6e745) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**